.CMake/compiler_opts.cmake

```diff
@@ -55,7 +55,7 @@ if(CMAKE_C_COMPILER_ID MATCHES "Clang|GNU")
         else()
             # Assume sensible default like -march=x86-64, -march=armv8-a, etc.
 	    if(ARCH_ARM64v8)
-                set(OQS_OPT_FLAG "-march=armv8-a+crypto")
+                set(OQS_OPT_FLAG "-march=armv8-a+crypto+sha3")
```

Comment:
Why is this not guarded/if'd by reference to ios ("APPLE"?) if it's indeed a problem only there?

Reply:
I didn't know if it is Apple specific. As I stated in my previous comment I wasn't able to find any documentation on the flags. Do you know where to find the documentation? Maybe this affects other vendors than Apple?

Reply:
I'll take this one back to draft and have a look at your suggestion in the related issue. Maybe I can get it to work with the toolchain file of this project.

Reply:
Hi @fwh-dc , this PR makes compilation on Cortex-A53 (as you can see in CI) failed. It does not have SHA3 extension.